contracts/minters/open-edition-minter/src/contract.rs

```diff
@@ -193,10 +200,20 @@ pub fn execute_purge(
     info: MessageInfo,
 ) -> Result<Response, ContractError> {
     nonpayable(&info)?;
-    // Check if mint has ended
+    // check if sold out (optional)
+    let mintable_num_tokens = MINTABLE_NUM_TOKENS.may_load(deps.storage)?;
```

Comment:
so what if it's ended but not sold out?

Reply:
`mintable_num_tokens` is only Some when there is no end time defined (as per instantiation), so the minting cannot be time limited in this case.